modules/ROOT/pages/access-source-message-attributes.adoc

```diff
@@ -0,0 +1,250 @@
+= Accessing Source Message Attributes
+
+When you create translation or reference maps for inbound or outbound message flows, you can access source message attributes in those maps. You can then map the attributes to fields in the translated message payload or use the attribute values in your mapping logic to perform dynamic condition execution.
+
+== Common Use Cases
+
+Accessing source message attributes helps in many use cases, such as with performing dynamic message routing decisions, performing specific functions, and mapping fields within the data transformation in the message flow execution.
+
+Some common use cases include:
+
+* Achieving end-to-end tracing
++
+Accessing transmission IDs in the translation maps enables you to achieve end-to-end tracing because these IDs are passed to the translated JSON or XML output file.
```

Comment:
```suggestion
Accessing transmission IDs in the translation maps enables end-to-end tracing because these IDs are passed to the translated JSON or XML output file.
```